cpu/native/include/gpiodev_linux.h

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (C) 2019 Benjamin Valentin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for
+ * more details.
+ */
+
+/**
+ * @defgroup    drivers_gpio_linux Linux User Mode GPIO Driver
+ * @ingroup     cpu_native
+ * @brief       Implementation of GPIO access from Linux User Space
+ *
+ * This module allows to connect a RIOT application that runs on a Linux host to
+ * the physical GPIO pins of that host. To do so, the application has to be
+ * compiled for the native board in a Linux environment.
+ *
+ * GPIO support is automatically included if either a module requiring the
+ * `PERIPH_GPIO` feature is added to the application or if it is explicitly
```

Comment:
The `periph_gpio` feature should be mentioned in lower-case.

```suggestion
 * `periph_gpio` feature is added to the application or if it is explicitly
``` 
You can squash directly.